### PR TITLE
Empty state in swagger editor

### PR DESCRIPTION
**Pull request by hwkr**
It's mostly working but I'm having some issues with reactivity when trying to set the example content (from the getting started tab).

https://github.com/scalar/scalar/assets/6374090/611f0c56-53ee-448b-b862-f7efda933adc

### CODE REVIEW

**Comment by marclave** [comment]
wow! such a huge improvement, excited to test this out after these meetings ✨

**Comment by hanspagel** [comment]
Love it

File: packages/swagger-editor/src/components/SwaggerEditor/SwaggerEditor.vue [hunk 1]
```diff
@@ -120,9 +116,9 @@ function handleChangeExample(example: GettingStartedExamples) {
     return
   }
 
-  handleSpecUpdate(spec)
-  currentExample.value = spec
-  rawContent.value = spec
+  activeTab.value = 'Swagger Editor'
+  await nextTick()
+  importHandler(spec)
```
Comment (hwkr) [inline packages/swagger-editor/src/components/SwaggerEditor/SwaggerEditor.vue:121]: @hanspagel I'm not quite sure how we should handle updating the spec here 🤔

**Comment by hanspagel** [comment]
Are we removing the tab then? 🤔 

Screenshot 2023-10-30 at 11 23 39

**Comment by hanspagel** [comment]
Let’s get this in and I refactor the data flow/reactivity stuff in a separate PR. ✌️

**Review by hanspagel — APPROVED** [review]

**Pull request merged**